packages/paste-core/components/tabs/src/Tab.tsx

```diff
@@ -37,21 +38,47 @@ const getTabBoxStyles = (orientation: Orientation, variant: Variants): BoxStyleP
           marginBottom: "space0",
         },
         _selected: {
-          borderLeftColor: isInverse ? "colorBorderInverseStrong" : "colorBorderPrimary",
           color: isInverse ? "colorTextInverse" : "colorTextLink",
         },
         _hover: {
-          borderLeftColor: isInverse ? "colorBorderInverse" : "colorBorderPrimaryStronger",
-          color: isInverse ? "colorTextInverseWeaker" : "colorTextLinkStronger",
+          color: isInverse ? "colorTextInverse" : "colorTextLinkStronger",
         },
         _disabled: {
           borderLeftColor: "transparent",
           color: isInverse ? "colorTextInverseWeakest" : "colorTextWeaker",
         },
         _focus: {
-          boxShadow: isInverse ? "shadowFocusInverse" : "shadowFocus",
+          boxShadow: isInverse ? "shadowFocusInverseInset" : "shadowFocusInset",
           outline: "none",
-          borderRadius: "borderRadius20",
+          borderTopLeftRadius: "borderRadius30",
+          borderBottomLeftRadius: "borderRadius30",
+        },
+        _selected_after: {
+          content: '" "',
+          position: "absolute",
+          height: "80%",
+          width: "4px",
+          backgroundColor: isInverse ? "colorBackgroundPrimaryWeakest" : "colorBackgroundPrimary",
+          borderTopLeftRadius: "borderRadius30",
+          borderBottomLeftRadius: "borderRadius30",
+          right: -1,
+          top: "10%",
+        },
+        _selected_hover_after: {
+          content: '" "',
+          position: "absolute",
+          height: "80%",
+          width: "4px",
+          backgroundColor: isInverse ? "colorBackgroundPrimary" : "colorBackgroundPrimaryStrongest",
+          borderTopLeftRadius: "borderRadius30",
+          borderBottomLeftRadius: "borderRadius30",
+          right: -1,
+        },
+        _focus_selected_after: {
+          display: "none",
+        },
+        _focus_selected_hover_after: {
+          display: "none",
         },
       };
     case "horizontal":
```

Comment:
chore: change inverse text color (default state) to `colorTextInverseWeaker`

Forgot to spec this out.

Reply:
nvm `colorTextInverseWeak`